api/types/role.go

```diff
@@ -176,6 +177,12 @@ type Role interface {
 	GetHostSudoers(RoleConditionType) []string
 	// SetHostSudoers sets the list of sudoers entries for the role
 	SetHostSudoers(RoleConditionType, []string)
+
+	// GetRequireSessionMFA returns true when all sessions in this cluster
+	// require an MFA check.
+	GetRequireSessionMFA() bool
```

Comment:
Why does this need to exist in `Role` and `RoleOptions`?

Reply:
`role.Options.RequireSessionMFA` is now a string rather than a bool, with possible values `on|off|hardware_key|hardware_key_touch`.

`role.GetRequireSessionMFA() bool` returns the actual value for `RequireSessionMFA`, by checking the role option:

```
// GetRequireSessionMFA returns true when all sessions in this cluster require
// an MFA check.
func (r *RoleV5) GetRequireSessionMFA() bool {
	switch r.Spec.Options.RequireSessionMFA {
	case constants.RequireSessionMFAOn, constants.RequireSessionMFAHardwareKey:
		return true
	default:
		return false
	}
}
```